src/gui/layout/qgslayoutpicturewidget.cpp

```diff
@@ -450,14 +330,39 @@ void QgsLayoutPictureWidget::setGuiElementValues()
       mAnchorPointComboBox->setEnabled( false );
     }
 
+    whileBlocking( mRadioSVG )->setChecked( mPicture->mode() == QgsLayoutItemPicture::FormatSVG );
```

Comment:
What happens if the mode is unknown?

Reply:
It shouldn't actually be possible -- unknown is now just a way of avoiding stable api breakage (i.e. scripts which don't explicitly set the mode)

Reply:
So when you load a pre 3.14 project, the picture items will automatically have their modes set to `QgsLayoutItemPicture::FormatSVG` ?

Reply:
Yep - it's done internally as soon as the picture source is set